picard/track.py

```diff
@@ -313,6 +313,9 @@ def _customize_metadata(self):
             tm['~silence'] = '1'
 
         if config.setting['use_genres']:
+            config = get_config()
```

Comment:
No need to set `config` here, it is already at start of this method.

Reply:
fixed in 5b50798